packages/cli/src/Server.ts

```diff
@@ -711,6 +721,63 @@ class App {
 			};
 		}));
 
+		// Retrieves all tags, with or without usage count
+		this.app.get(`/${this.restEndpoint}/tags`, ResponseHelper.send(async (req: express.Request, res: express.Response): Promise<IShortTag[] | Array<IShortTag & UsageCount>> => {
+			if (req.query.withUsageCount === 'true') {
+				const tablePrefix = config.get('database.tablePrefix');
+				const tagsWithCountDb = await TagHelpers.getTagsWithCountDb(tablePrefix);
+				return tagsWithCountDb.map(({ id, ...rest }) => ({ id: id.toString(), ...rest }));
+			}
+
+			const tags = await Db.collections.Tag!.find({ select: ['id', 'name'] });
+
+			return tags.map(TagHelpers.shortenTag);
+		}));
+
+		// Creates a tag
+		this.app.post(`/${this.restEndpoint}/tags`, ResponseHelper.send(async (req: express.Request, res: express.Response): Promise<IShortTag | void> => {
```

Comment:
maybe we should add hooks to all these new tags endpoints similar to the workflow ones